README.md

```diff
@@ -2,6 +2,8 @@
 
 [![Donated by Ranjit](https://img.shields.io/badge/donated%20by-@WhatsARanjit-fb7047.svg)](#transfer-notice)
 
+We would like to inform you that the tag v1.0.0 was mistakenly created on the HEAD commit, but the module was released with an earlier commit. To address this, we have released a new version, v1.0.1.
```

Comment:
maybe also mention that the git tag was deleted again?

Reply:
resolving as we have updated with new content.